cap-ci/create_cap-release_pipeline.sh

```diff
@@ -1,28 +1,40 @@
 #!/bin/bash
-set -x
```

Comment:
should we keep xtrace?

Reply:
I don't think we should keep `xtrace` in scripts by default.